site/content/docs/main/config/request-routing.md

```diff
@@ -117,6 +122,10 @@ For `header` conditions there is one required field, `name`, six operator fields
 
 - `regex` is a string representing a regular expression, and checks that the header value matches against the given regular expression.
 
+Modifiers:
+- `ignoreCase`: IgnoreCase specifies that string matching should be case insensitive. It has no effect on the `Regex` parameter.
+- `TreatMissingAsEmpty`: specifies if the header match rule specified header does not exist, this header value will be treated as empty. Defaults to false. Unlike the underlying Envoy implementation this is **only** supported for negative matches (e.g. NotContains, NotExact).
```

Comment:
```suggestion
- `treatMissingAsEmpty`: specifies if the header match rule specified header does not exist, this header value will be treated as empty. Defaults to false. Unlike the underlying Envoy implementation this is **only** supported for negative matches (e.g. NotContains, NotExact).
```

Reply:
fixed in bdd8b20a58135f00906b651196b6240e56dba593